src/sv-pipeline/05_annotation/scripts/compute_AFs.py

```diff
@@ -199,54 +330,87 @@ def main():
     
     #Add relevant fields to header
     INFO_ADD = [
-    '##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles genotyped (for biallelic sites) or individuals with copy-state estimates (for multiallelic sites).">',
-    '##INFO=<ID=AC,Number=A,Type=Integer,Description="Number of non-reference alleles observed (for biallelic sites) or individuals at each copy state (for multiallelic sites).">',
-    '##INFO=<ID=AF,Number=A,Type=Float,Description="Allele frequency (for biallelic sites) or copy-state frequency (for multiallelic sites).">',
+    '##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles genotyped (biallelic sites only).">',
+    '##INFO=<ID=AC,Number=A,Type=Integer,Description="Number of non-reference alleles observed (biallelic sites only).">',
+    '##INFO=<ID=AF,Number=A,Type=Float,Description="Allele frequency (biallelic sites only).">',
     '##INFO=<ID=N_BI_GENOS,Number=1,Type=Integer,Description="Total number of individuals with complete genotypes (biallelic sites only).">',
     '##INFO=<ID=N_HOMREF,Number=1,Type=Integer,Description="Number of individuals with homozygous reference genotypes (biallelic sites only).">',
     '##INFO=<ID=N_HET,Number=1,Type=Integer,Description="Number of individuals with heterozygous genotypes (biallelic sites only).">',
     '##INFO=<ID=N_HOMALT,Number=1,Type=Integer,Description="Number of individuals with homozygous alternate genotypes (biallelic sites only).">',
     '##INFO=<ID=FREQ_HOMREF,Number=1,Type=Float,Description="Homozygous reference genotype frequency (biallelic sites only).">',
     '##INFO=<ID=FREQ_HET,Number=1,Type=Float,Description="Heterozygous genotype frequency (biallelic sites only).">',
-    '##INFO=<ID=FREQ_HOMALT,Number=1,Type=Float,Description="Homozygous alternate genotype frequency (biallelic sites only).">'
+    '##INFO=<ID=FREQ_HOMALT,Number=1,Type=Float,Description="Homozygous alternate genotype frequency (biallelic sites only).">',
+    '##INFO=<ID=CN_NUMBER,Number=1,Type=Integer,Description="Total number of individuals with estimated copy numbers (multiallelic CNVs only).">',
+    '##INFO=<ID=CN_COUNT,Number=.,Type=Integer,Description="Number of individuals observed at each copy state, starting from CN=0 (multiallelic CNVs only).">',
+    '##INFO=<ID=CN_FREQ,Number=.,Type=Float,Description="Frequency of individuals observed at each copy state, starting from CN=0 (multiallelic CNVs only).">',
+    '##INFO=<ID=CN_NONREF_COUNT,Number=1,Type=Integer,Description="Sum of all individuals with non-reference copy states (multiallelic CNVs only).">',
+    '##INFO=<ID=CN_NONREF_FREQ,Number=1,Type=Float,Description="Total frequency of all individuals across all non-reference copy states (multiallelic CNVs only).">'
```

Comment:
To make the descriptions of these two tags a little more consistent, I'd change them to,

```
'##INFO=<ID=CN_NONREF_COUNT,Number=1,Type=Integer,Description="Number of samples with non-reference copy states (multiallelic CNVs only).">'
`##INFO=<ID=CN_NONREF_FREQ,Number=1,Type=Float,Description="Frequency of samples with non-reference copy states (multiallelic CNVs only).">'
```

Reply:
Good point—done.